src/NServiceBus.AcceptanceTesting/Support/ScenarioRunner.cs

```diff
@@ -358,11 +358,16 @@ static string GenerateTestTimedOutMessage(TimeSpan maxTime)
                 runners.Add(runner);
             }
 
-            await Task.WhenAll(runners.Select(r => r.InitializeTask).ToArray()).ConfigureAwait(false);
-            var failedRunner = runners.FirstOrDefault(r => r.InitializeTask.Result.Failed);
-            if (failedRunner != null)
+            var tasks = runners.Select(r => r.InitializeTask).ToArray();
+            await Task.WhenAll(tasks).ConfigureAwait(false);
+
+            foreach (var runner in runners)
             {
-                throw new ScenarioException($"Endpoint {failedRunner.Instance.Name()} failed to initialize", failedRunner.InitializeTask.Result.Exception);
+                var result = await runner.InitializeTask;
```

Comment:
yeah i thought about this one. my brain triggers when ever i see `.Result` so i went with the async. so if u want to go back to the result that is fine. but perhaps add a comment?